client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -126,3 +136,20 @@ module.exports = React.createClass( {
 		);
 	}
 } );
+
```

Comment:
>There is still quite a bit of non-es6 syntax in this file

Similar to remarks at https://github.com/Automattic/wp-calypso/pull/3197#discussion_r52539734, I'll probably leave the remaining legacy for future pull requests.